.travis.yml

```diff
@@ -27,7 +27,8 @@ install:
   - pip install --force-reinstall pylint==1.9.2
 
 script:
-  - pylint adafruit_gfx.py
+  - pylint adafruit_gfx/*.py
+  - pylint adafruit_gfx/fonts/*.py
```

Comment:
Perfect! Thank you for adding these lines to this file!

Reply:
my pleasure